easybuild/easyconfigs/x/XGBoost/XGBoost-1.6.0-foss-2021a-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'CMakePythonPackage'
+
+name = 'XGBoost'
+version = '1.6.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://github.com/dmlc/xgboost'
+description = """XGBoost is an optimized distributed gradient boosting library designed to be highly efficient,
+ flexible and portable."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+sources = [{
+    'filename': 'xgboost.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/dmlc',
+        'repo_name': 'xgboost',
+        'tag': 'v1.6.0',
+        'recursive': True,
+        'keep_git_dir': True,
+    },
+}]
+
+# Can't add proper SHA256 checksum, because source tarball is created locally after recursive 'git clone'
+checksums = [None]
+
+builddependencies = [
+    ('CMake', '3.20.1'),
+    ('Python', '3.9.5'),
+]
+
+dependencies = [
+    ('CUDA', '11.3.1', '', SYSTEM),
+    ('SciPy-bundle', '2021.05'),
+]
```

Comment:
consider adding `NCCL` as a dependency as well, for distributed GPU support:
https://xgboost.readthedocs.io/en/stable/build.html#building-with-gpu-support

Reply:
rebuilding now with NCCL support to see if that works

Reply:
Rebuild worked, however I don't know how to test if it actually works.